Keas.Mvc/ClientApp/components/Spaces/SpacesTable.tsx

```diff
@@ -58,89 +58,90 @@ const workstationFilter = (rows: any[], id, filterValue) => {
 const getRowWorkstationsInUse = (row: any) => row.original?.workstationsInUse;
 const getRowWorkstationsTotal = (row: any) => row.original?.workstationsTotal;
 
-export default class SpacesTable extends React.Component<IProps, {}> {
-  public render() {
-    const columns: Column<ISpaceInfo>[] = [
-      {
-        Cell: data => (
-          <Button
-            color='link'
-            onClick={() => this.props.showDetails(data.row.original.space)}
-          >
-            Details
-          </Button>
-        ),
-        Header: ' ',
-        maxWidth: 150
+const SpacesTable = (props: IProps) => {
+  // export default class SpacesTable extends React.Component<IProps, {}> {
+  const columns: Column<ISpaceInfo>[] = [
```

Comment:
I would have thought you needed to do a useMemo() for the columns here?

Reply:
I think we added it for the EquipmentTable since there was an issue on it before but I can add it in now.

Reply:
ok give it a shot -- my understanding is that as a class there isn't a problem but w/ a function + hooks you don't want the whole column object redefined on each load so memo is used.